alibi_detect/cd/base.py

```diff
@@ -19,7 +19,7 @@
 logger = logging.getLogger(__name__)
 
 
-class BaseClassifierDrift(BaseDetector, DriftConfigMixin):
+class BaseClassifierDrift(BaseDetector):
```

Comment:
Is the intended paradigm to inherit from the `DriftConfigMixin` at the highest (public class) level only?

Reply:
Yes, in order to limit duplication, the convention is to inherit `DriftConfigMixin` in the relevent `cd.base` class.

However, this is complicated by the way we implement the "dispatch" wrapper classers for back-end specific detectors such as `MMDDrift`. These don't inherit from a base class so we have to inherit `DriftConfigMixin` there instead. Inheriting in `MMDDriftTF` etc would also complicate things since their kwargs are different to their public class.

Think we'd need to write up conventions here in `CONTRIBUTING.md`...